prowler/providers/kubernetes/exceptions/exceptions.py

```diff
@@ -0,0 +1,73 @@
+from prowler.exceptions.exceptions import ProwlerException
+
+
+class KubernetesBaseException(ProwlerException):
+    """Base class for Kubernetes errors."""
+
+    KUBERNETES_ERROR_CODES = {
+        (1925, "KubernetesCloudResourceManagerAPINotUsedError"): {
+            "message": "Cloud Resource Manager API not used",
+            "remediation": "Enable the Cloud Resource Manager API for the project.",
+            "file": "{file}",
+            "provider": "Kubernetes",
```

Comment:
Do we need to have always the `"provider": "Kubernetes",` key? I think we can get it from the exception class.